support-workers/src/main/scala/com/gu/emailservices/DigitalPackEmailFields.scala

```diff
@@ -2,106 +2,81 @@ package com.gu.emailservices
 
 import com.gu.emailservices.SubscriptionEmailFieldHelpers._
 import com.gu.i18n.Currency
-import com.gu.salesforce.Salesforce.SfContactId
 import com.gu.support.promotions.Promotion
 import com.gu.support.workers._
 import com.gu.support.workers.states.PaymentMethodWithSchedule
 
-// Output Json should look like this:
```

Comment:
I don't know if this JSON was still in date but better to have it in a test than a comment.